akka-persistence/src/main/scala/akka/persistence/PersistentActor.scala

```diff
@@ -37,47 +40,52 @@ case object RecoveryCompleted extends RecoveryCompleted {
  * @param replayMax maximum number of messages to replay. Default is no limit.
  */
 @SerialVersionUID(1L)
-final case class Recover(fromSnapshot: SnapshotSelectionCriteria = SnapshotSelectionCriteria.Latest, toSequenceNr: Long = Long.MaxValue, replayMax: Long = Long.MaxValue)
+final case class Recovery(
+  fromSnapshot: SnapshotSelectionCriteria = SnapshotSelectionCriteria.Latest,
+  toSequenceNr: Long = Long.MaxValue,
+  replayMax: Long = Long.MaxValue)
+
+object Recovery {
 
-object Recover {
   /**
-   * Java API.
-   *
-   * @see [[Recover]]
+   * Java API
+   * @see [[Recovery]]
    */
-  def create() = Recover()
+  def create() = Recovery()
 
   /**
-   * Java API.
-   *
-   * @see [[Recover]]
+   * Java API
+   * @see [[Recovery]]
    */
   def create(toSequenceNr: Long) =
-    Recover(toSequenceNr = toSequenceNr)
+    Recovery(toSequenceNr = toSequenceNr)
 
   /**
-   * Java API.
-   *
-   * @see [[Recover]]
+   * Java API
+   * @see [[Recovery]]
    */
   def create(fromSnapshot: SnapshotSelectionCriteria) =
-    Recover(fromSnapshot = fromSnapshot)
+    Recovery(fromSnapshot = fromSnapshot)
 
   /**
-   * Java API.
-   *
-   * @see [[Recover]]
+   * Java API
+   * @see [[Recovery]]
    */
   def create(fromSnapshot: SnapshotSelectionCriteria, toSequenceNr: Long) =
-    Recover(fromSnapshot, toSequenceNr)
+    Recovery(fromSnapshot, toSequenceNr)
 
   /**
-   * Java API.
-   *
-   * @see [[Recover]]
+   * Java API
+   * @see [[Recovery]]
    */
   def create(fromSnapshot: SnapshotSelectionCriteria, toSequenceNr: Long, replayMax: Long) =
-    Recover(fromSnapshot, toSequenceNr, replayMax)
+    Recovery(fromSnapshot, toSequenceNr, replayMax)
+
+  /**
+   * Convenience method for skipping recovery in [[PersistentActor]].
+   * @see [[Recovery]]
+   */
+  def none(): Recovery = Recovery(toSequenceNr = 0L)
```

Comment:
sorry @2beaucoup, forgot that one - amended now!
// I'm a bit sick and not up to full capacity...